src/handlers/public/send-transfer.js

```diff
@@ -16,7 +16,12 @@ module.exports = ({ plugin, prefix, routingTable, internalUri, uuidSecret, ilpEr
   switch (type) {
     case IlpPacket.Type.TYPE_ILP_PAYMENT:
       nextHop = routingTable.getNextHop(data.account)
-      nextAmount = nextHop.curveRemote.amountReverse(data.amount)
+      if (nextHop.local) {
```

Comment:
it would be cleaner to seed the curveCache with identity function curves for local routes, than to have this special case of .local, but that's maybe something that can be done in a follow-up